docs/api/apiv3/paths/workspaces_schema.yml

```
@@ -0,0 +1,156 @@
+# /api/v3/workspaces/schemas
+---
+get:
+  responses:
+    '200':
+      content:
+        application/hal+json:
+          examples:
+            response:
+              value:
+                _dependencies: []
+                _attributeGroups:
+                  - _type: ProjectFormCustomFieldSection
+                    name: Project Details
```

Should probably use workspace in the example names? Not sure about this one, as users will never create/edit actual, generic workspaces. So for this case, an example using any of these types might be better 🤔 But the other examples in this file use `workspace`, so to stay consistent:


```suggestion
                    name: Workspace Details
```